Neo4j.Driver/Neo4j.Driver.Tests/Mapping/ConventionTranslation/FormatterTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) "Neo4j"
+// Neo4j Sweden AB [https://neo4j.com]
+// 
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using FluentAssertions;
+using Neo4j.Driver.Mapping.ConventionTranslation;
+using Xunit;
+
+namespace Neo4j.Driver.Tests.Mapping.ConventionTranslation;
+
+public class FormatterTests
+{
+    [Fact]
+    public void SnakeCaseCombiner_ShouldFormat()
```

Comment:
```suggestion
    public void SnakeCaseFormatter_ShouldFormat()
```

Nitpick. The Formatter and Identifier tests still use "extractor" and "combiner". Not a blocker so I will approve, up to you if you feel the need to amend.